include/picongpu/plugins/output/WriteSpeciesCommon.hpp

```diff
@@ -145,16 +145,20 @@ struct FreeMemory
         if (ptr != nullptr)
         {
 #if( PMACC_CUDA_ENABLED == 1 )
-            auto rc = cudaFreeHost(ptr);
-            /* cupla can't handle foreign memory allocated with `cudaHostAlloc`
-             * therefore the cupla error `cuplaErrorMemoryAllocation` is ignored
-             */
-            if(rc != cuplaErrorMemoryAllocation)
-                CUDA_CHECK(rc)
+/* cupla is not supporting the function cudaHostAlloc to create mapped memory.
```

Comment:
can you please open a cupla & alpaka issue and link it here in the comment?

Even if it might currently be a `wontfix` it needs to be doc-ed as an issue.

Reply:
I can open a feature request in alpaka. 
This bug is not important for cupla because I as user ignored the error cupla provided.

Reply:
I opened the alpaka feature request: https://github.com/ComputationalRadiationPhysics/alpaka/issues/612

Reply:
thx, please link it in both comments in-code :)

Reply:
Please link https://github.com/ComputationalRadiationPhysics/alpaka/issues/296 as the original alpaka issue

Reply:
typo: `cupla 0.1.0 does not support ...`